### PR TITLE
Joseph/create chat

### PR DESCRIPTION
**Pull request by aledecos**
Create Chat complete.

-Adding to Mongo
-Error checking
-User Display
-returning to lobby once complete

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -80,6 +80,84 @@ io.on('connection', function(socket){
     }); 
   });
 
+
+  //Get User list
+  socket.on('get users for create chat', function(obj){
+    //Check if a user already exists with the submitted email
```
Comment (SaraK207) [inline server.js:86]: Update or delete this comment

**Review by SaraK207 — APPROVED** [review]
Looks good to me. I would suggest reviewing the comments as some of them are for other sections and haven't been removed/updated for the new code. Another change is to add a label for the chat name text box. At first it wasn't clear to me that I was supposed to enter the chat name there.
Also, currently we are only able to choose one user and one moderator for the chat. If we get time we should add functionality to choose multiple users/moderators.

**Comment by farfar336** [comment]
Suggest adding the following:
-Add "Chat Name" label beside chat name field
-Display message "Hold ctrl while clicking users to select multiple users"

**Comment by farfar336** [comment]
> Also, currently we are only able to choose one user and one moderator for the chat. If we get time we should add functionality to choose multiple users/moderators.

It already exists, it's just not intuitive. The user can hold ctrl and click multiple users to select them. Joseph and I were thinking it would be easier to display a message saying "Hold ctrl while clicking users to select multiple users" to make the user aware of this functionality.

What do you think?

File: server.js (hunk 1 above)
Comment (farfar336) [reply server.js:86]: I suggest updating the code in lines 86, 138 and 147:

image

**Comment by farfar336** [comment]
Joseph won't be able to work on this until next week. He and I agree that I can go to his branch and implement the suggestions above.

**Pull request closed without merging**